src/styles/mixins/layout.scss

```diff
@@ -78,3 +78,13 @@
     margin-top: var.$layout-margin;
   }
 }
+
+/**
+ * Establish vertical space outside of elements in the container
+ *
+ * @param $size [var.$layout-margin--medium]: Spacing size (padding)
+ */
+@mixin vertical-space($size: var.$layout-margin--medium) {
```

Comment:
@LMS007 has raised a good point that the naming of the underlying variable here (involving "margin") should be rethought to be something more generic, like "spacing." It's on my list.

Reply:
👍